run_tests_CLI/array_api_run_tests.py

```diff
@@ -84,7 +84,7 @@ def main():
             test, backend = line.split(",")
             backend = backend.strip("\n")
             coll, submod, test_fn = get_submodule(test)
-            command = f'docker run --rm --env IVY_BACKEND={backend} --env ARRAY_API_TESTS_MODULE="ivy" --env REDIS_URL={redis_url} --env REDIS_PASSWD={redis_pass} -v "$(pwd)":/ivy -v "$(pwd)"/.hypothesis:/.hypothesis unifyai/ivy:latest timeout 30m python3 -m pytest {test} -k \"{k_flag[backend]}\" --tb=short -vv'  # noqa
+            command = f'docker run --rm --env IVY_BACKEND={backend} --env ARRAY_API_TESTS_MODULE="ivy" --env REDIS_URL={redis_url} --env REDIS_PASSWD={redis_pass} -v "$(pwd)":/ivy -v "$(pwd)"/.hypothesis:/.hypothesis unifyai/ivy:latest timeout 30m python3 -m pytest {test} -k "{k_flag[backend]}" --tb=short -vv'  # noqa
```

Comment:
kindly revert all the changes unrelated to your implementation. confirm that the unit tests are passing before we proceed. thanks